plumbing/storer/shallow.go

```diff
@@ -0,0 +1,10 @@
+package storer
+
+import "gopkg.in/src-d/go-git.v4/plumbing"
+
+// ShallowStorer storage of shallow commits, meaning that it does not have the
+// parents of a commit (explanation from git documentation)
+type ShallowStorer interface {
```

Comment:
Why the interfaces repeat the word `Storer` in their name when it is already part of the package name?